settings.d/openscap.yml.example

```diff
@@ -22,3 +22,7 @@
 # Directory where OpenSCAP report XML are stored
 # In case sending to Foreman succeeded, yet failed to save to reportsdir
 #:failed_dir: /usr/share/foreman-proxy/openscap/failed
+
+# Directory where corrupted OpenSCAP report XML are stored
+# When proxy cannot parse the report sent by client
+#:corrupted_dir: /usr/share/foreman-proxy/openscap/corrupted
```

Comment:
I know it's an example, but why don't you use ```/var/lib/foreman-proxy/openscap/corrupted``` (like in the puppet module) here?